### PR TITLE
Add stage for running tests for python scripts

### PR DESCRIPTION
**Pull request by krdln**

### CODE REVIEW

**Comment by krdln** [comment]
recheck no bug

**Comment by krdln** [comment]
recheck no bug

File: Jenkinsfile [hunk 1]
```diff
@@ -61,6 +62,20 @@ pipeline {
             }
         }
 
+        stage('Linux test') {
```
Comment (jacek-izykowski) [inline Jenkinsfile]: MJ - there was some issue with stage names containing spaces. We were removing them recently.

File: Jenkinsfile [hunk 2]
```diff
@@ -61,6 +62,20 @@ pipeline {
...
+            when { expression { env.ghprbPullId } }
```
Comment (jacek-izykowski) [inline Jenkinsfile]: This stage is going to be executed for all repositories from GitHub, which is not necessary (including https://github.com/Juniper/contrail-windows).

**Review by jacek-izykowski — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (krdln) [reply Jenkinsfile]: The goal was just to make gerrit logs cleaner and don't let random people wonder "why are linux tests running in windows CI". I don't think `contrail-windows` would be a problem here.

**Review by jacek-izykowski — APPROVED** [review]

**Review by jablonskim — APPROVED** [review]

**Pull request merged**